staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -445,7 +445,7 @@ func NewConfig(codecs serializer.CodecFactory) *Config {
 		// Generic API servers have no inherent long-running subresources
 		LongRunningFunc:                     genericfilters.BasicLongRunningRequestCheck(sets.NewString("watch"), sets.NewString()),
 		lifecycleSignals:                    lifecycleSignals,
-		StorageObjectCountTracker:           flowcontrolrequest.NewStorageObjectCountTracker(),
+		StorageObjectCountTracker:           nil,
```

Comment:
this breaks apiservers other than kube-apiserver, doesn't it?

Reply:
@sttts thanks, I have reverted this change and added comment.